examples/main.go

```diff
@@ -35,7 +44,18 @@ func main() {
 			return
 		}
 		// Log raw message struct
-		fmt.Println(mes)
+		//fmt.Println(mes)
```

Comment:
though I am not sure how we would handle signaling out then.. maybe bot.Start() could return a channel indicating a quit